packages/components/psammead-media-player/src/index.stories.jsx

```diff
@@ -11,6 +14,12 @@ const withDuration = {
   datetime: 'PT2M30S',
 };
 
+const StyledMessageContainer = styled.div`
+  padding-top: 56.25%;
```

Comment:
Just wondering what 56.25% is for? This value seems unsual?

Reply:
It's a means of applying a 16:9 ratio to the container's padding. Where 16 is the width, 56.26% of that (9) is the height.

We use it in Simorgh as a means of pushing the other page content below the player - 

<img width="780" alt="Screenshot 2020-08-18 at 12 28 18" src="https://user-images.githubusercontent.com/9042672/90507751-53ed9800-e14e-11ea-81c1-989dce990b91.png">

Frankly it seems like a mad way to manage layout but I'm sure there's good reason for doing it.

Reply:
Intersting, ok thanks for the explanation @RichardPK